requirements-docker.txt

```diff
@@ -6,4 +6,5 @@ scikit-image==0.24.0
 matplotlib==3.9.2
 numpy==2.1.0
 pandas==2.2.2
-pyspark==3.4.0
\ No newline at end of file
+pyspark==3.4.0
+requests==2.32.3
```

Comment:
Not related to this specific commit, but the == in requirements should probably be relaxed to >= in most cases (since == will be very rigid/inflexible outside of a container or virtual environment). Will note this in a separate issue.